marklogic-data-hub-central/ui/src/components/raw-data/raw-data-card-view/raw-data-card-view.tsx

```diff
@@ -1,76 +1,110 @@
+      fontSize: sourceFmt === 'json' ? '8px' : '8.5px',
+      borderRadius: '50%',
+      textAlign: 'center',
+      color: '#ffffff',
+      verticalAlign: 'middle',
+      marginRight: '8px'
     }
+    return customStyles;
+  }
 
-    const displayUri = (uri) => {
-        return getLastChars(uri, 20, '...');
-    }
+  const displayUri = (uri) => {
+    return formatCardUri(uri)
+  }
 
-    return (
-        <div id="raw-data-card" aria-label="raw-data-card" className={styles.rawDataCard}>
-            <Row gutter={24} type="flex" >
-                {props.data && props.data.length > 0 ? props.data.map((elem, index) => (
-                    <Col key={index}>
-                        <div>
-                            <Card
-                                className={styles.cardStyle}
-                                size="small"
-                            >
-                                <div className={styles.cardMetadataContainer}>
-                                    <span className={styles.uriContainer} data-testid={elem.uri + '-URI'}>URI: <span className={styles.uri}>{displayUri(elem.uri)}</span></span>
-                                    <span className={styles.cardIcons}>
-                                        <span><Icon type="info-circle" className={styles.infoIcon} theme="filled" data-testid={elem.uri + '-InfoIcon'} /></span>
-                                        <span className={styles.sourceFormat}
-                                            style={sourceFormatStyle(elem.format)}
-                                            data-testid={elem.uri + '-sourceFormat'}
-                                        >{sourceFormatOptions[elem.format].label}</span>
-                                        <MLTooltip title={'Show the raw source data'} placement="top"
-                                        ><i role="detail-link icon" data-testid={elem.uri + '-detailViewIcon'}><FontAwesomeIcon icon={faExternalLinkAlt} className={styles.detailLinkIcon} size="lg" /></i>
-                                        </MLTooltip>
-                                    </span>
-                                </div>
-                            </Card>
-                        </div>
-                    </Col>)) : <span></span>}
-            </Row>
+  const displaySnippet = (item) => {
+    if (['json', 'xml', 'text'].includes(item.format)) {
+      let str = '';
+      item.matches.forEach(item => {
+        item['match-text'].forEach(element => {
+          if (typeof element === 'object') {
+            str = str.concat('<b>').concat(element.highlight).concat('</b>').concat('...')
+          } else {
+            str = str.concat(element)
+          }
+        });
+      })
+      return <p>{ReactHtmlParser(str)}</p>;
+    } else if ('binary' === item.format) {
+      return (
+        <div className={styles.binaryCard} >
+          <FileOutlined style={{ fontSize: '24px', color: '#777' }} />
```

Comment:
Let's place the style in the corresponding module.scss file and then refer it here using the className property. 
It should always be placed in scss file unless the style is customized in a way that scss file cannot process it.

Reply:
Fixed. Thanks.